Userland/Libraries/LibCpp/Tests/parser/strace.cpp

```diff
@@ -135,7 +135,7 @@ int main(int argc, char** argv)
             arg3,
             res);
 
-        if (!trace_file->write(string)) {
+        if (trace_file->write(string).is_error()) {
```

Comment:
It seems this file is used in testing LibCpp and depends on the exact contents.
You probably need to regenerate the `Userland/Libraries/LibCpp/Tests/parser/strace.ast` file for the tests to pass.
I have quickly looked around but am not sure how to do that.
Perhaps cc @itamar8910 knows more about this

Reply:
Sure, try running `cpp-parser /usr/src/serenity/Libraries/LibCpp/Tests/parser/strace.cpp > strace.ast` and then replace `Userland/Libraries/LibCpp/Tests/parser/strace.ast` with the result.

Reply:
So, I tried doing this and could only get it to work inside of Serenity itself. This would be fine if I had any clue as to how to export the resulting file - after a lot of different attempts, I ended up trying to send it through the internet - pastebin it, email it to myself, and so on - but the browser wasn't really up for the task. 

Honestly, at this point I find my enthusiasm for this project almost non-existant; I think it would be for the best if I were to simply abandon this PR. I was working on cleaning up the KeyboardMapper when I got distracted by this. I think that I'll continue work on that instead

Reply:
It's too bad that you had this experience. Next time please feel free to talk about technical problems you're having on Discord.

The method I use for transferring files out of serenity is `mount`ing the _disk_image on the host.

Reply:
Discord is a little intimidating though :^)

I did try to mount it but I couldn't get it to work. I have no idea why - I just tried again and this time it worked just fine. Imma just blame the fact that it's linux